client/src/App.js

```diff
@@ -7,7 +7,7 @@ import TeacherForm from "./pages/TeacherForm";
 
 const App = () => (
 	<Routes>
-		<Route path="/" element={<Home />} />
```

Comment:
Hi , can you please advise why this was deleted and why it was changed to the below

Reply:
fix this already check the new merge code